documentation/src/pages/prompt-library/data/prompts/pr-impact-analyzer.json

```diff
@@ -0,0 +1,29 @@
+  "extensions": [
+    {
+      "name": "GitHub",
+      "command": "npx -y @modelcontextprotocol/server-github",
```

Comment:
Thanks for the contribution! Quick note

this is an old / archived MCP. Have you tried this prompt out? 

This is the actual MCP server https://github.com/github/github-mcp-server